src/test/java/org/cactoos/iterator/JoinedTest.java

```diff
@@ -42,11 +43,13 @@ public void joinsIterators() {
         MatcherAssert.assertThat(
             "Can't concatenate mapped iterators together",
             new LengthOf(
-                new NoNulls<>(
-                    new Joined<Iterator<String>>(
-                        new Mapped<>(
-                            input -> new IteratorOf<>(input),
-                            new IteratorOf<>("x")
+                new org.cactoos.iterable.NoNulls<>(
+                    new IterableOf<>(
+                        new Joined<Iterator<String>>(
```

Comment:
@mazdack is the explicit typing really needed?

Reply:
yes, otherwise we get a warning 

> unchecked generic array creation for varargs parameter of type...